lib/Default/MysqlProperties.class.php

```diff
@@ -0,0 +1,82 @@
+<?php declare(strict_types=1);
```

Comment:
I think this file needs to be deleted now that you have `MySqlProperties`.

Reply:
Hmm, weird. I did rename it. Unfortunately, I think because Windows is not case-sensitive, it isn't tracking correctly in git. I will try to fix it.

Reply:
I had to set `git config core.ignorecase false`, then change the name a little, then change it back and it triggered a delete for `MysqlProperties.class.php`. I think it's fixed -- good catch